tests/rptest/tests/controller_upgrade_test.py

```diff
@@ -26,11 +26,11 @@
 
     # < 22.2 versions may log bare std::exception error
     # (https://github.com/redpanda-data/redpanda/issues/5886)
-    re.compile("rpc - .*std::exception"),
```

Comment:
Is this a direct effect from the server refactor? If so, i'm curious about the allowed logs that supposedly only exist in older versions. If they've been problematic, maybe we should update the comment to point to an issue that still exists?

Reply:
This one didn't need updating really -- it's only here for upgrade tests that run 22.1.

The update is harmless but unnecessary.

Reply:
thanks for flagging @andrwng i didn't analyze the cases in depth since it's a level of allowed-ness that wasn't really being tested explicitly.